tests/precompile/solidity/suites.go

```diff
@@ -22,35 +22,35 @@ var _ = ginkgo.Describe("[Precompiles]", ginkgo.Ordered, func() {
 		ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
 		defer cancel()
 
-		utils.RunDefaultHardhatTests(ctx, "contract_native_minter")
+		utils.RunDefaultHardhatTests(ctx, utils.SubnetsIDs["contract_native_minter"], "contract_native_minter")
```

Comment:
The variable that needs to be passed in here is the `blockchainID` as opposed to the `subnetID`, could we change the name of this variable?